app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
It's not a good practice to use mutable types as class variables because they are shared between all instances of the class. This can lead to unexpected behavior. In your case, it seems like you're using it to store all created `Person` instances. A better way to do this would be to manage this storage outside of the `Person` class, for example in a separate `PersonManager` class or in the global scope of your module.

Reply:
в іншому випадку виникають помилки